lnclient/lnd/lnd.go

```diff
@@ -444,6 +447,12 @@ func NewLNDService(ctx context.Context, eventPublisher events.EventPublisher, ln
 				default:
 					payment, err := paymentStream.Recv()
 					if err != nil {
+						if grpcErr, ok := status.FromError(err); ok {
+							if grpcErr.Code() == codes.Unknown {
```

Comment:
`codes.Unknown` feels wrong, but we should otherwise use `grpcErr.Message() == "routerrpc server shutting down"` which looks hacky, what should I do here?